channels/pe.m3u

```diff
@@ -139,5 +139,9 @@ https://query-streamlink.lanesh4d0w.repl.co/iptv-query?streaming-ip=https://www.
 https://5ee0faac3bbae.streamlock.net:443/viatv2021/viatv2021/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/StereotvOficial/picture?width=320&height=320" group-title="Music",Stereo TV (Peru) [Not 24/7]
 https://servers.amelbasoluciones.co:19360/5medialive/5medialive.m3u8
-#EXTINF:-1 tvg-id="BestCableSportsPeru.pe" tvg-name="Best Cable Sports Perú" tvg-country="PE" tvg-language="" tvg-logo="" group-title="",Best Cable Sports Perú (720p) [Not 24/7]
+#EXTINF:-1 tvg-id="BestCableSportsPeru.pe" tvg-name="Best Cable Sports Perú" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/BestCableSportsCanal6/picture?width=320&height=320" group-title="Sports",Best Cable Sports Perú (720p) [Not 24/7]
 https://live.siete.us/bestcablesports/bestcablesports/index.m3u8
+#EXTINF:-1 tvg-id="BestCableMusic.pe" tvg-name="Best Cable Music" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/BestCableMusicCanal97/picture?width=320&height=320" group-title="Music",Best Cable Music (720p) [Not 24/7]
+https://live.siete.us/bestcablesports/bestcablesports/index.m3u8
```

Comment:
Are you sure that's the correct link you've put there ?

Reply:
<!--StartFragment-->
masterManifest | https://live.siete.us/bestcablemusic/bestcablemusic/index.m3u8
-- | --
mediaManifest | https://live.siete.us/bestcablemusic/bestcablemusic/tracks-v1a1/mono.m3u8

<!--EndFragment-->